main/acle.md

```diff
@@ -2837,14 +2838,11 @@ the selection algorithm is platform dependent, where with platform means
 CPU/Vendor/OS as in the target triplet.
 2. The selection is permanent for the
 lifetime of the process.
-3. Only those versions could be considered where all
-dependent features are available.
-
-Rules of version selection are in order:
-
-4. Select the most specific version else
-5. select the version with the highest priority else
-6. `"default"` is selected if no other versions are suitable.
+3. Among any two versions, the higher priority version is determined by
+identifying the highest priority feature that is specified in exactly one of
+the versions, and selecting that version.
+4. The selection algorithm must select the highest priority versions whose
```

Comment:
Do the current priority numbers reflect anything meaningful at the moment? Is it worth converting them to powers of two so that we can create bitmasks to represent version priorities including dependent features?

Reply:
The numbers don't (and never did) convey any useful meaning beyond helping to indicate an ordering. That column in the table could possibly be removed, but it would then need some words adding to explain which end of the table is highest priority.

I think writing explicit powers of two would be actively unhelpful, because either they wouldn't be stable, or they would require large gaps. This doesn't preclude using a bitmask within a compiler - in fact, that seems like a good way to implement the selection algorithm.